From c65dc6b46933a9de4e33f84d8107d2548ebda105 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Fri, 14 Nov 2014 13:50:15 +0800
Subject: [PATCH 1077/1094] fsl-echi:fix a compile warning

Fixed a compile warning introduced by fsl-ehci driver:

In file included from drivers/usb/host/ehci-hcd.c:1247:0:
drivers/usb/host/ehci-fsl.c: In function 'usb_phy_clk_valid':
drivers/usb/host/ehci-fsl.c:58:16: warning: unused variable 'non_ehci' [-Wunused-variable]
  void __iomem *non_ehci = hcd->regs;
                ^
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/usb/host/ehci-fsl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/ehci-fsl.c b/drivers/usb/host/ehci-fsl.c
index e3d71ae..8bde4ea 100644
--- a/drivers/usb/host/ehci-fsl.c
+++ b/drivers/usb/host/ehci-fsl.c
@@ -55,9 +55,9 @@ struct ehci_fsl {
 
 static bool usb_phy_clk_valid(struct usb_hcd *hcd)
 {
-	void __iomem *non_ehci = hcd->regs;
 	bool ret = true;
 #ifndef CONFIG_CRASH_DUMP
+	void __iomem *non_ehci = hcd->regs;
 	if (!(in_be32(non_ehci + FSL_SOC_USB_CTRL) & PHY_CLK_VALID))
 		ret = false;
 #endif
-- 
2.0.2

